subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_rest

```diff
@@ -19,20 +19,6 @@ config NRF_CLOUD_REST_FRAGMENT_SIZE
 	range 128 1700
 	default 1700
 
-config NRF_CLOUD_REST_SEND_TIMEOUT
-	int "Socket send timeout, in seconds"
-	default 60
-	help
-	  Sets the timeout duration in seconds to use when sending data.
-	  To disable, set the timeout duration to -1.
-
-config NRF_CLOUD_REST_RECV_TIMEOUT
-	int "Socket receive timeout, in seconds"
-	default 60
-	help
-	  Sets the timeout duration in seconds to use when receiving data.
-	  To disable, set the timeout duration to -1.
-
```

Comment:
Please, check / update where used. Otherwise, will break the builds, at least MoSH and multicell lib builds are.